src/apps/explorer/components/TransanctionBatchGraph/styled.ts

```diff
@@ -25,7 +25,7 @@ export function STYLESHEET(theme: DefaultTheme): Stylesheet[] {
         width: 2,
         'target-arrow-shape': 'triangle',
         'target-arrow-color': theme.grey,
-        'curve-style': 'unbundled-bezier',
+        'curve-style': 'bezier',
```

Comment:
Not sure about this change:
![image](https://user-images.githubusercontent.com/622217/166259129-864f10c2-c69b-456b-99e5-0e8c734c95a6.png)
Before (with: `'curve-style': 'unbundled-bezier'` )
![image](https://user-images.githubusercontent.com/622217/166259441-3b742b5c-4824-4c3d-b206-183e6f81c947.png)

Reply:
Yes it looks worse but without it you keep having the same reported issue. Could you see if there's a way to make them look similar? Maybe we could suggest merging the 2 arrows into 1 as we discussed.